examples/tutorials/translations/bengali/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<ul><li>In other languages, we've been putting the translators section below the "Authors" section. I think it should go right under "লেখক: Andrew Trask - Twitter: <a href="https://twitter.com/iamtrask" target="_blank">@iamtrask</a>" </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2938/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
ok. missed that. redoing it.

Reply:
fixed it